src/platform/stub.c

```diff
@@ -0,0 +1,76 @@
+/* General module of stub of functions that have not been decompiled */
```

Comment:
Amazing how small this is :D

Reply:
Yeah, insane how much we have accomplished! :D